Odata-docs/webapi-8/getting-started.md

```diff
@@ -0,0 +1,381 @@
+
+## Build the Edm model and configure the service
+Replace the code in the _Startup.cs_ class with the following:
+```csharp
+namespace Lab01
+{
+    using Lab01.Models;
+    using Microsoft.AspNetCore.Builder;
+    using Microsoft.AspNetCore.OData;
+    using Microsoft.Extensions.DependencyInjection;
+    using Microsoft.OData.ModelBuilder;
+
+    public class Startup
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            var modelBuilder = new ODataConventionModelBuilder();
+            modelBuilder.EntityType<Order>();
+            modelBuilder.EntitySet<Customer>("Customers");
+
+            services.AddControllers().AddOData(
+                options => options.Select().Filter().OrderBy().Expand().Count().SetMaxTop(null).AddRouteComponents(
+                    "odata",
+                    modelBuilder.GetEdmModel()));
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            app.UseRouting();
+
+            app.UseEndpoints(endpoints => endpoints.MapControllers());
+        }
+    }
+}
+```
+In the _Startup.cs_ class, we define the Edm model. As part of the model, we register `Order` as an entity type and `Customers` as an entity set - `Customer` entity type also gets registered as a result. In this case, the `ODataConventionalModelBuilder` is being used to build the Edm model.
+
+We then proceed to add essential OData services by calling the `AddOData` method, in the process enabling different OData query capabilities - `$select`, `$filter`, `$orderby`, `$expand`, `count`, `$top` and `$skip`. The `AddRouteComponents` method is used to register a route passing along the Edm model to associate it with.
+
+The final step in configuring the service is adding the two middlewares responsible for matching incoming HTTP requests and dispatching those requests to the application's executable endpoints. We do this by calling `UseRouting` and `UseEndpoints` - the former adds route matching to the middleware pipeline while the latter adds endpoint execution.
+
+
+## Run the OData service
+Before running the service, configure the debugging profile to use Kestrel web server by selecting the profile named after the project - **Lab01**
+
+![Screenshot of configuring debugging profile in visual studio](../assets/2022-11-11-aspnet-core-odata-8-getting-started-debugging-profile.png)
+
+Press F5 to build and run the application.
+
+The following window will be displayed:
+
+![Screenshot of running aspnet core odata 8 application](../assets/2022-11-11-aspnet-core-odata-8-getting-started-run-app.png)
+
+## Interact with the OData service from an API client
+You can use an API client of your choice to interact with the OData service.
```

Comment:
Should we mention the `$odata` debug endpoint or is that too much detail for a start?

Reply:
I think it would probably be better to include it in the CRUD article to avoid overloading the beginner with too much detail.

Reply:
@habbes I agree with @asanyaga about not including it in this tutorial. It'd be better to cover that in the tutorial for routing because basically what the `$odata` endpoint does is provide someone with a view of the exposed endpoints.